lib/cjs/generated/rum.d.ts

```diff
@@ -151,6 +151,50 @@ export declare type RumActionEvent = CommonProperties & ViewContainerSchema & {
     };
     [k: string]: unknown;
 };
+/**
+ * Schema of all properties of an Transition event
+ */
+export declare type RumTransitionEvent = CommonProperties & StreamContainerSchema & {
+    /**
+     * RUM event type
+     */
+    readonly type: 'transition';
+    /**
+     * Transition properties
+     */
+    readonly transition: {
+        /**
+         * Type of the transition
+         */
+        readonly type: string;
+        /**
+         * UUID of the transition
+         */
+        readonly id?: string;
+        /**
+         * Timestamp in the player
+         */
+        readonly ts?: number;
```

Comment:
Likely a question for the SDK team, should we stay consistent with other events and rename this to `timestamp`?